cluster/up.sh

```diff
@@ -5,8 +5,14 @@ set -ex
 source ./cluster/cluster.sh
 cluster::install
 
+export KUBVIRT_WITH_CNAO_SKIP_CONFIG=true
```

Comment:
KUBEVIRT. But I see it's a typo in kubevirtci itself, won't block on it.

Reply:
damnn ><